### PR TITLE
New package: light-locker-1.6.0

### PR DESCRIPTION
**Pull request by LockeAnarchist**

### CODE REVIEW

File: srcpkgs/light-locker/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+pkgname="light-locker"
+version="1.6.0"
+revision=1
+build_style=gnu-configure
+short_desc="A simple session-locker for lightdm"
+maintainer="Emanuel Serpa <emanuel@openmailbox.org>"
+hostmakedepends="pkg-config"
+makedepends="intltool libglib-devel libX11-devel gtk+3-devel dbus-glib-devel"
+depends="lightdm"
+license="GPLv3"
```
Comment (beefcurtains) [inline srcpkgs/light-locker/template]: ``GPL-3`` (it has to be a file in ``/usr/share/licenses``)

File: srcpkgs/light-locker/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'light-locker'
+
+pkgname="light-locker"
+version="1.6.0"
+revision=1
+build_style=gnu-configure
+short_desc="A simple session-locker for lightdm"
+maintainer="Emanuel Serpa <emanuel@openmailbox.org>"
+hostmakedepends="pkg-config"
+makedepends="intltool libglib-devel libX11-devel gtk+3-devel dbus-glib-devel"
```
Comment (beefcurtains) [inline srcpkgs/light-locker/template]: ``libglib-devel`` is redundant, dependency of ``dbus-glib-devel``

File: srcpkgs/light-locker/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'light-locker'
+
+pkgname="light-locker"
+version="1.6.0"
+revision=1
+build_style=gnu-configure
+short_desc="A simple session-locker for lightdm"
+maintainer="Emanuel Serpa <emanuel@openmailbox.org>"
+hostmakedepends="pkg-config"
+makedepends="intltool libX11-devel gtk+3-devel dbus-glib-devel"
```
Comment (xtraeme) [inline srcpkgs/light-locker/template]: - `intltool` must be in `hostmakedepends` instead.

**Comment by xtraeme** [comment]
The build now fails to cross compile because it needs `glib-devel` in `hostmakedepends`.

**Comment by xtraeme** [comment]
musl is broken, but it's not your fault.

**Pull request merged**